src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/button.scss

```diff
@@ -1,12 +1,60 @@
 @import '../../containers/Application/colors.scss';
 
-$buttonTextColor: $blueZodiac;
-$buttonTextColorActive: $shakespeare;
-$buttonTextColorDisabled: $silver;
-$buttonActiveBackgroundColor: $wildSand;
+$buttonTextLightColor: $blueZodiac;
+$buttonTextDarkColor: $concrete;
+$buttonTextLightColorActive: $shakespeare;
+$buttonTextDarkColorActive: $white;
+$buttonTextLightColorDisabled: $silver;
+$buttonTextDarkColorDisabled: $scorpion;
+$buttonActiveBackgroundLightColor: $wildSand;
+$buttonActiveBackgroundDarkColor: $mirage;
 $primaryTextColor: $white;
 $primaryBackgroundColor: $shakespeare;
 
+.light {
+    color: $buttonTextLightColor;
+
+    &:disabled {
+        color: $buttonTextLightColorDisabled;
+    }
+
+    &:not(:disabled) {
+        &:hover,
+        &.active {
+            color: $buttonTextLightColorActive;
+            background-color: $buttonActiveBackgroundLightColor;
+        }
+    }
+
+    &.primary,
+    &.primary:hover {
+        color: $primaryTextColor;
+        background-color: $primaryBackgroundColor;
+    }
+}
+
+.dark {
+    color: $buttonTextDarkColor;
+
+    &:disabled {
+        color: $buttonTextDarkColorDisabled;
+    }
+
+    &:not(:disabled) {
+        &:hover,
+        &.active {
+            color: $buttonTextDarkColorActive;
+            background-color: $buttonActiveBackgroundDarkColor;
+        }
+    }
+
+    &.primary,
+    &.primary:hover {
+        color: $primaryTextColor;
+        background-color: $primaryBackgroundColor;
+    }
```

Comment:
If the definition for `primary` is the both for same styles I would hoist it up to the root level instead of duplicating it resp. move it inside the root button.